packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -186,6 +217,7 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
       if (!torrent.done && wire.amChoking) {
         wire._clearTimeout();
       }
+      this.informListeners();
```

Comment:
I am not sure if this is needed. Does the UI need to be updated on these events?

Reply:
I would say safest to say 'yes', especially given (as we discusssed) as updating the react tree is cheap, and react will decide whether anything actually changed. 

This event *might* impact on the "Number of Peers connected" UI, and it might also be something we want to display in future as it indicates that our peer is still live (an important concept for us).